### ISSUE TITLE
Download OSCAL SSP link produces error instead of OSCAL SSP on upgraded versions of GovReady

### ISSUE THREAD

**Issue by gregelin** [GovReady/govready-q#1345]
## Bug description
OSCAL SSP link downloaded content describes and error instead of showing an OSCAL SSP in JSON.

This issue is for the OSCAL SSP JSON download not working after a migration of the database of an existing install of GovReady to version 0.9.1.50.1.  (We have fixed one error in the OSCAL SSP JSON download (Bug #1344).)

## Legacy install bug
- branch `feature/oscal-components-1.0.0-rc1` or branch `feature/v0.9.1.50.1`
- Lightweight_ATO_template/apps/light-ato-apps v 0.2.9
-  /q-files/vendors/govready/govready-q-files-startpack/q-files/lightweight-ato/ v 1.0.0 (diff shows v 0.2.9 and v 1.0.0 to be the same)

A second error on an legacy, but upgraded, install of GovReady-q, downloading a draft SSP in OSCAL json produces an error with the following text displayed in the downloaded `sip_v1_oscal_json.json` file:

```
Reverse for 'profile_oscal_json' with keyword arguments '{'system_id': 84}' not found. 2 pattern(s) tried: ['controls/(?P<system_id>.)/profile/oscal/json', 'systems/(?P<system_id>.)/profile/oscal/json']
```

## Debugging Help Needed
@woodt It is possible there was a database change to support OSCAL SSP JSON?

## To Reproduce
The error only occurs after upgrading a legacy version of GovReady to `v0.9.1.50.1`. But we are not sure what might be in the database causing the issue.

## Expected behavior
OSCAL SSP is downloaded

**Comment by gregelin** [comment]
I moved the database from my the directory with my “upgraded” version, and then ran the ./install-govready-q-sh script to create a NEW database for v0.9.1.50.1 in that directory.

And the downloading of OSCAL now works.

That suggests there is an issue with upgrading the databases that is causing the OSCAL SSP to not work. Something not loaded in the data, a migration missing.

The link triggering the process from a `light-ato-ssp` IT System questionnaire's project's SSP to download the OSCAL SSP in JSON is `http://localhost:8000/tasks/215/system-security-plan/download/document/ssp_v1_oscal_json/json` where `215` is the value of the task (not the system). 

This is the line in the `module_logic` that reverse is used being reported in the error. However, this is 

```
# generate a URL to reference this system's OSCAL profile (baseline) profile_path = reverse('profile_oscal_json', kwargs=dict(system_id=system.id))
```
if the system isn't there it will fail system id.

Could it be that there is not accurate linkage in the database to get from the task to the system id in older versions of the database?

**Comment by gregelin** [comment]
Fixed in 0.9.1.50.2 #1350.
